### DISCUSSION TITLE
Historical Price Data

### DISCUSSION THREAD

**Discussion by aenurgy** [ercot/api-specs#51] [Q&A]
Currently, the historical price data on ERCOT's website only goes as far back as 2010 as far as I can tell. 

I am interested in accessing historical ERCOT price data for the years 2001 to 2009, specifically:
- DAM clearing prices for capacity (NP4-188-CD)
- DAM Settlement Point Prices (NP4-190-CD)
- Settlement Point Prices at Resource Nodes, Hubs and Load Zones (NP6-905-CD)

I am open to accessing such data through API or simple csv file downloads from the ERCOT website.

Please point me in the right direction.

**Comment by CaffeineLab** [comment]
I hope you get lucky and someone will just zip up the data for you.  I don't go back that far myself.  Otherwise, you are going to have to most likely fill out a request with an ERCOT Account Manager...   

Looking at gridstatus.io - they also only go back to 2010....

**Reply by CaffeineLab** [reply]
I have heard that since the Nodal Market began in 2010, this is pretty much it from this source.   Let us know if you find anything though.

**Reply by mackermann-ercot** [reply]
CaffeineLab, 

You are absolutely correct.  ERCOT transitioned to the Nodal network model on December 1, 2010.  Prior ("Zonal" network) information is not available.

Regards,
-mackermann, ERCOT
